dali/operators/random/uniform.cc

```diff
@@ -19,12 +19,20 @@ namespace dali {
 
 void Uniform::AssignRange(HostWorkspace &ws) {
   auto &output = ws.OutputRef<CPUBackend>(0);
+  if (range_[0] >= range_[1]) {
+    for (int i = 0; i < batch_size_; ++i) {
+      sample_data[k] = range_[0];
+    }
+    return;
+  }
```

Comment:
It's no becessary if we enfornce that the range is not empty.

Reply:
Done